From 2bb01c3a20f47eccb85aac6fdc128f0702ffe974 Mon Sep 17 00:00:00 2001
From: Alicia Sparklejoy <joy@example.com>
Date: Thu, 25 Aug 2022 22:35:10 +0200
Subject: [PATCH 1/2] Adapt page for mobile devices

Add some media queries and make the CSS definitions mobile-first. Fit
into viewport on mobile devices.
---
 index.html |  1 +
 style.css  | 19 +++++++++++++++++--
 2 files changed, 18 insertions(+), 2 deletions(-)

diff --git a/index.html b/index.html
index 41e4893..e4295ef 100644
--- a/index.html
+++ b/index.html
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <title>We love Puppies!</title>
 	<link rel="stylesheet" href="style.css" />
+	<meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
 <body>
 	<header>
diff --git a/style.css b/style.css
index fb0e71d..191d073 100644
--- a/style.css
+++ b/style.css
@@ -1,5 +1,6 @@
 img {
-	max-height: 300px;
+	max-height: 250px;
+	max-width: 280px;
 }
 
 header h1 {
@@ -10,7 +11,7 @@ header h1 {
 main {
 	display: flex;
 	flex-wrap: wrap;
-	margin: 20px 80px;
+	margin: 10px 5px;
 }
 
 .puppy-box {
@@ -18,6 +19,20 @@ main {
 	justify-content: center;
 	margin: 10px;
 	box-sizing: border-box;
+	text-align: center;
+
+	/* just for debugging how big these containers are, remove when done */
+	background: green;
+}
+
+@media screen and (min-width: 720px) {
+  img {
+	  max-height: 300px;
+	  max-width: unset;
+  }
+  main {
+	margin: 20px 80px;
+  }
 }
 
 
-- 
2.37.2

